cmake/extensions.cmake

```diff
@@ -68,6 +72,10 @@ function(target_cddl_source target cddl_file)
     set(type_file_arg --oht ${h_file_dir}/${CDDL_TYPE_FILE_NAME})
   endif()
 
+  if (DEFINED CDDL_DEFAULT_MAXQ)
```

Comment:
I don't think you need the `DEFINED` here, IMO the code looks better without it.